nms/app/packages/magmalte/app/views/traffic/PolicyEdit.js

```diff
@@ -95,154 +85,127 @@ export default function PolicyRuleEditDialog(props: Props) {
-        <Tab key="policy" data-testid="epcTab" label={'Policy'} />
-        <Tab key="flows" data-testid="ranTab" label={'Flows'} />
-        <Tab key="tracking" data-testid="trackingTab" label={'Tracking'} />
-        <Tab key="redirect" data-testid="redirectTab" label={'Redirect'} />
-        <Tab key="app" data-testid="appTab" label={'App'} />
-        <Tab key="qos" data-testid="qosTab" label={'QoS'} />
-        <Tab
-          key="subscribers"
-          data-testid="subsribersTab"
-          label={'Subscribers'}
-        />
+        {tabList.map(tabKey => (
+          <Tab key={tabKey} data-testid={tabKey + 'Tab'} label={tabKey} />
+        ))}
       </Tabs>
-      {tabPos === 0 && (
-        <PolicyInfoEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          isNetworkWide={isNetworkWide}
-          setIsNetworkWide={setIsNetworkWide}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
-      {tabPos === 1 && (
-        <PolicyFlowsEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
-      {tabPos === 2 && (
-        <PolicyTrackingEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
-      {tabPos === 3 && (
-        <PolicyRedirectEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
-      {tabPos === 4 && (
-        <PolicyAppEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
-      {tabPos === 5 && (
-        <PolicyQosEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          qosProfiles={qosProfiles}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
-      {tabPos === 6 && (
-        <PolicySubscribersEdit
-          policyRule={rule}
-          onChange={(policyRule: policy_rule) => setRule(policyRule)}
-          subscribers={props.subscribers}
-          descriptionClass={classes.description}
-          dialogClass={classes.dialog}
-          inputClass={classes.inputField}
-        />
-      )}
```

Comment:
Why is this being removed?

Reply:
I'm removing this tab because this is not a part of this workflow. As shown here
https://www.figma.com/file/hw8iheccicYs6tW7gsFsPK/Magma---Client-Facing-Final-Design?node-id=361%3A65933

It is more part of this
https://www.figma.com/proto/aRJzvSa3J9nhD2n3yERd1s/Magma---Phase-2-(Updated-Styles)?node-id=1005%3A107&scaling=min-zoom
which is being tracked by this PR -> https://github.com/magma/magma/issues/2840
Even if we plan to add the subscriber tab, It must more be aligned with this subscriber tab shown above, so i just left it out for now and thought of including it when we do that.